packages/db/src/index.ts

```diff
@@ -1,11 +1,264 @@
+ *   - `"contracts"`
+ *     [_**named**_; _resource_: [[DataModel.Contract | Contract]];
+ *     _input_: [[DataModel.ContractInput | ContractInput]]]
+ *   - `"contractInstances"`
+ *     [_resource_: [[DataModel.ContractInstance | ContractInstance]];
+ *     _input_: [[DataModel.ContractInstanceInput | ContractInstanceInput]]]
+ *   - `"nameRecords"`
+ *     [_resource_: [[DataModel.NameRecord | NameRecord]];
+ *     _input_: [[DataModel.NameRecordInput | NameRecordInput]]]
+ *   - `"networks"`
+ *     [_**named**_; _resource_: [[DataModel.Network | Network]];
+ *     _input_: [[DataModel.NetworkInput | NetworkInput]]]
+ *   - `"networkGenealogies"`
+ *     [_resource_: [[DataModel.NetworkGenealogy | NetworkGenealogy]];
+ *     _input_: [[DataModel.NetworkGenealogyInput | NetworkGenealogyInput]]]
+ *   - `"projectNames"`
+ *     [_**mutable**_; _resource_: [[DataModel.ProjectName | ProjectName]];
+ *     _input_: [[DataModel.ProjectNameInput | ProjectNameInput]]]
+ *   - `"projects"`
+ *     [_resource_: [[DataModel.Project | Project]];
+ *     _input_: [[DataModel.ProjectInput | ProjectInput]]]
+ *   - `"sources"`
+ *     [_resource_: [[DataModel.Source | Source]];
+ *     _input_: [[DataModel.SourceInput | SourceInput]]]
+ *
+ * This list is not intended to be static; since @truffle/db is in early
+ * release, it may make sense to add new collections / change relationships
+ * between existing collections. Backwards compatibility is planned but not yet
+ * guaranteed.
+ *
+ * ## Other interfaces
+ *
+ * ### JavaScript / TypeScript interface
+ *
+ * This package exposes programmatic interfaces for working with
+ * the resources listed above:
+ *   - [[Process.resources]], a set of four generator functions that encode
+ *     logic for storing and retrieving resources for a given `collectionName`.
+ *
+ *   - [[Process.Run.forDb]], to construct an `async` helper that facilitates
+ *     requests/responses from/to the above generator functions against a given
+ *     [[Db]] instance.
+ *
+ * In addition, please see the [[Resources]] module for handy helper types for
+ * dealing with @truffle/db entities.
+ *
+ * ### Truffle-specific interface
+ *
+ * This package also provides an abstraction to interface with other
+ * Truffle data formats, namely `WorkflowCompileResult`, returned by
+ * \@truffle/workflow-compile, and the Truffle contract artifacts format,
+ * defined by @truffle/contract-schema. This abstraction covers two classes:
+ *   - [[Project.Project]] for operations that **do not** require a network
+ *     connection. Use function [[Project.initialize | `Project.initialize()`]]
+ *     to create.
+ *   - [[Project.ConnectedProject]] for operations that **do** require a
+ *     blockchain network. Use existing project abstraction's
+ *     [[Project.connect | `project.connect()`]] method to create.
+ *
+ * ### HTTP interface
+ *
+ * This package exposes the [[serve | `serve()`]] function, which returns an
```

Comment:
This is a bit unclear. I suggest clarifying that `serve` is part of a truffle command that can be used to access this interface.

Reply:
From the perspective of the @truffle/db package, `serve` is just another interface. If people want to use @truffle/db without Truffle, they'll likely want to use this function on its own.

Any ideas on making this more clear?